proposals/simd/webassembly-opcodes.md

```diff
@@ -0,0 +1,279 @@
+|:---------------------------------------|:--------------|
+| `v128.and(a: v128, b: v128) -> v128`   | [v128.and](portable-simd.md#bitwise-operations) |
+| `v128.or(a: v128, b: v128) -> v128`    | [v128.or](portable-simd.md#bitwise-operations) |
+| `v128.xor(a: v128, b: v128) -> v128`   | [v128.xor](portable-simd.md#bitwise-operations) |
+| `v128.not(a: v128) -> v128`            | [v128.not](portable-simd.md#bitwise-operations) |
+| `v128.load(addr, offset) -> v128`      | [v128.load](portable-simd.md#load) |
+| `v128.store(addr, offset, data: v128)` | [v128.store](portable-simd.md#store) |
+
+## `v8x16` operations
+| WebAssembly                                                 | Portable SIMD |
+|:------------------------------------------------------------|:--------------|
+| `v8x16.select(s: b8x16, t: v128, f: v128) -> v128`          | [v8x16.select](portable-simd.md#lane-wise-select) |
+| `v8x16.swizzle(a: v128, s: LaneIdx16[16]) -> v128`          | [v8x16.swizzle](portable-simd.md#swizzle-lanes) |
+| `v8x16.shuffle(a: v128, b: v128, s: LaneIdx32[16]) -> v128` | [v8x16.shuffle](portable-simd.md#shuffle-lanes) |
+
+## `i8x16` operations
+| WebAssembly                                                | Portable SIMD |
+|:-----------------------------------------------------------|:--------------|
+| `i8x16.build(x: i32[16]) -> v128`                          | [i8x16.build](portable-simd.md#build-vector-from-individual-lanes) |
+| `i8x16.splat(x: i32) -> v128`                              | [i8x16.splat](portable-simd.md#create-vector-with-identical-lanes) |
+| `i8x16.extractLane_s(a: v128, i: LaneIdx16) -> i32`        | [i8x16.extractLane](portable-simd.md#extract-lane-as-a-scalar) |
+| `i8x16.extractLane_u(a: v128, i: LaneIdx16) -> i32`        | [i8x16.extractLane](portable-simd.md#extract-lane-as-a-scalar) |
+| `i8x16.replaceLane(a: v128, i: LaneIdx16, x: i32) -> v128` | [i8x16.replaceLane](portable-simd.md#replace-lane-value) |
+| `i8x16.add(a: v128, b: v128) -> v128`                      | [i8x16.add](portable-simd.md#integer-addition) |
+| `i8x16.sub(a: v128, b: v128) -> v128`                      | [i8x16.sub](portable-simd.md#integer-subtraction) |
+| `i8x16.mul(a: v128, b: v128) -> v128`                      | [i8x16.mul](portable-simd.md#integer-multiplication) |
+| `i8x16.neg(a: v128) -> v128`                               | [i8x16.neg](portable-simd.md#integer-negation) |
+| `i8x16.addSaturate_s(a: v128, b: v128) -> v128`            | [s8x16.addSaturate](portable-simd.md#saturating-integer-addition) |
+| `i8x16.addSaturate_u(a: v128, b: v128) -> v128`            | [u8x16.addSaturate](portable-simd.md#saturating-integer-addition) |
+| `i8x16.subSaturate_s(a: v128, b: v128) -> v128`            | [s8x16.subSaturate](portable-simd.md#saturating-integer-subtraction) |
+| `i8x16.subSaturate_u(a: v128, b: v128) -> v128`            | [u8x16.subSaturate](portable-simd.md#saturating-integer-subtraction) |
```

Comment:
I agree saturation is useful. I'm asking if the first version needs this though. Not necessarily the first ship of SIMD, simply the first proposal.

I ask because by adding saturation here you're making the portable performance case that needs to be built harder, in my opinion.